### PR TITLE
update getting-started tutorial

### LINKED ISSUE #879: typo in getting-started input usage
https://github.com/tektoncd/triggers/blob/045e704ce44d75e274e2490e3c1d98041dc74f60/docs/getting-started/pipeline.yaml#L74

I was getting an error until I changed this to `"$(resources.inputs.image-source.url)"`

### LINKED ISSUE #902: Getting started eventlisteners RBAC issues
# Expected Behavior
Eventlistener pod run with no errors

# Actual Behavior
Eventlistener started with following error 

```
E0115 04:30:22.504882       1 reflector.go:178] k8s.io/client-go@v11.0.1-0.20190805182717-6502b5e7b1b5+incompatible/tools/cache/reflector.go:125: Failed to list *v1alpha1.ClusterTriggerBinding: clustertriggerbindings.triggers.tekton.dev is forbidden: User "system:serviceaccount:getting-started:tekton-triggers-admin" cannot list resource "clustertriggerbindings" in API group "triggers.tekton.dev" at the cluster scope
E0115 04:30:22.505056       1 reflector.go:178] k8s.io/client-go@v11.0.1-0.20190805182717-6502b5e7b1b5+incompatible/tools/cache/reflector.go:125: Failed to list *v1alpha1.EventListener: eventlisteners.triggers.tekton.dev is forbidden: User "system:serviceaccount:getting-started:tekton-triggers-admin" cannot list resource "eventlisteners" in API group "triggers.tekton.dev" in the namespace "getting-started"
E0115 04:30:22.505281       1 reflector.go:178] k8s.io/client-go@v11.0.1-0.20190805182717-6502b5e7b1b5+incompatible/tools/cache/reflector.go:125: Failed to list *v1alpha1.TriggerBinding: triggerbindings.triggers.tekton.dev is forbidden: User "system:serviceaccount:getting-started:tekton-triggers-admin" cannot list resource "triggerbindings" in API group "triggers.tekton.dev" in the namespace "getting-started"
E0115 04:30:22.505418       1 reflector.go:178] k8s.io/client-go@v11.0.1-0.20190805182717-6502b5e7b1b5+incompatible/tools/cache/reflector.go:125: Failed to list *v1alpha1.TriggerTemplate: triggertemplates.triggers.tekton.dev is forbidden: User "system:serviceaccount:getting-started:tekton-triggers-admin" cannot list resource "triggertemplates" in API group "triggers.tekton.dev" in the namespace "getting-started"
E0115 04:30:22.505652       1 reflector.go:178] k8s.io/client-go@v11.0.1-0.20190805182717-6502b5e7b1b5+incompatible/tools/cache/reflector.go:125: Failed to list *v1alpha1.Trigger: triggers.triggers.tekton.dev is forbidden: User "system:serviceaccount:getting-started:tekton-triggers-admin" cannot list resource "triggers" in API group "triggers.tekton.dev" in the namespace "getting-started"
...
```

# Steps to Reproduce the Problem
I am following the instructions from the guides

https://github.com/tektoncd/triggers/tree/master/docs/getting-started
https://github.com/tektoncd/pipeline/blob/master/docs/install.md

1. minikube start --memory 6144 --cpus 2 --insecure-registry="127.0.0.1:5000"
2. minikube addons enable registry
3. minikube addons enable registry-aliases
4. minikube addons enable ingress
5. eval $(minikube -p minikube docker-env)
6. kubectl apply --filename https://storage.googleapis.com/tekton-releases/pipeline/latest/release.yaml
7. kubectl create clusterrole tutorial-role --verb=* --resource=deployments,deployments.apps
8. kubectl create clusterrolebinding tutorial-binding --clusterrole=tutorial-role --serviceaccount=default:tutorial-service
9. kubectl apply --filename https://storage.googleapis.com/tekton-releases/triggers/latest/release.yaml
10. kubectl create secret docker-registry regcred --docker-server=https://registry-1.docker.io/v1/ --docker-username=sample --docker-password=****** --docker-email=sample@sample.com
11. kubectl create namespace getting-started
12. kubectl apply -f ./docs/getting-started/rbac/admin-role.yaml
13. kubectl apply -f ./docs/getting-started/rbac/webhook-role.yaml
14. kubectl apply -f ./docs/getting-started/pipeline.yaml
15. kubectl apply -f ./docs/getting-started/triggers.yaml # with modified DOCKER IMAGE replace
16. k logs -n getting-started el-getting-started-listener* # shows above error

# Additional Info

- Kubernetes version:

  **Output of `kubectl version`:**

```
Client Version: version.Info{Major:"1", Minor:"17+", GitVersion:"v1.17.14-dispatcher", GitCommit:"766e62361387c284022a97270f0997d67ae792d0", GitTreeState:"clean", BuildDate:"2020-12-01T16:13:56Z", GoVersion:"go1.13.9", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"20", GitVersion:"v1.20.0", GitCommit:"af46c47ce925f4c4ad5cc8d1fca46c7b77d13b38", GitTreeState:"clean", BuildDate:"2020-12-08T17:51:19Z", GoVersion:"go1.15.5", Compiler:"gc", Platform:"linux/amd64"} 
```

- Tekton Pipeline version:

  **Output of `tkn version` or `kubectl get pods -n tekton-pipelines -l app=tekton-pipelines-controller -o=jsonpath='{.items[0].metadata.labels.version}'`**

```
Client version: 0.13.0
Pipeline version: v0.20.0
Triggers version: v0.10.2
```

### PR DESCRIPTION
**Pull request by dibyom**
# Changes

Fixes for old syntax, and RBAC role changes.

Fixes #902 , fixes #879

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#tests) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#docs) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commits)
- [ ] Release notes block has been filled in or deleted (only if no user facing changes)

_See [the contribution guide](https://github.com/tektoncd/triggers/blob/master/CONTRIBUTING.md) for more details._

# Release Notes

```release-note
NONE
```

### CODE REVIEW

File: docs/getting-started/README.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ your cluster, you will need this for your Webhook in later steps.
 - Create the _getting-started_ namespace, where all our resources will live.
   - `kubectl create namespace getting-started`
 - [Create the admin user, role and rolebinding](./rbac/admin-role.yaml)
-  - `kubectl apply -f ./docs/getting-started/rbac/admin-role.yaml`
+  - `kubectl -n getting-started apply -f ./docs/getting-started/rbac/admin-role.yaml`
```
Comment (MarcelMue) [inline docs/getting-started/README.md:43]: Can we set the default either consistently inside the file or when applying? Currently roughly half the files have the namespace hardcoded while the other half is applied like this.

Comment (dibyom) [reply docs/getting-started/README.md:43]: done.

**Review by khrm — COMMENTED** [review]
/approve

**Pull request merged**